### PR TITLE
validation: support int64 int32 int16 and int8 type on 64-bit platform

### PR DESCRIPTION
**Pull request by miraclesu**
#2658

### CODE REVIEW

**Comment by astaxie** [comment]
/home/travis/gopath/src/github.com/astaxie/beego/validation/validators.go:173:12: unnecessary conversion
/home/travis/gopath/src/github.com/astaxie/beego/validation/validators.go:215:12: unnecessary conversion

**Comment by miraclesu** [comment]
Has been fixed.

File: validation/validation.go [hunk 1]
```diff
@@ -144,17 +144,17 @@ func (v *Validation) Required(obj interface{}, key string) *Result {
 }
 
 // Min Test that the obj is greater than min if obj's type is int
-func (v *Validation) Min(obj interface{}, min int, key string) *Result {
+func (v *Validation) Min(obj interface{}, min int64, key string) *Result {
```
Comment (astaxie) [inline validation/validation.go]: It's a break API change

Comment (miraclesu) [inline validation/validation.go]: Has been fixed.

**Pull request merged**